neurom/io/tests/test_utils.py

```diff
@@ -172,16 +170,6 @@ def test_load_neuromorpho_3pt_soma():
     _check_neurites_have_no_parent(nrn)
 
 
-NRN = utils.load_neuron(FILENAMES[0])
-
-
-def test_neuron_section_ids():
```

Comment:
we don't need to test this anymore?

Reply:
Section id are already tested in plenty of places in these unit tests. 
See for example, all the tests in https://github.com/BlueBrain/NeuroM/blob/c9a222dd59bff1e71166e12a20899a50e7af4006/neurom/core/tests/test_iter.py#L109

It does not need to be tested once more, especially in an unrelated file called `test_utils.py`.